server/server.py

```diff
@@ -0,0 +1,11 @@
+from flask import Flask
+
+app = Flask(__name__)
+
+# Example api route
+@app.route("/members")
+def members():
+  return {"members": ["Member1", "Member2", "Member3"]}
```

Comment:
do we need to commit an example route if this is not what we'll be using? I think sample code we could leave along with PR comments to reference if we need to